invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml

```diff
@@ -57,7 +57,7 @@
   props:
     csl: chapter
     datacite_general: Text
-    datacite_type: Book section
+    datacite_type: Book chapter
```

Comment:
Changing the information related to the resource type, however, I'm keeping the `id` for backwards compatibility. The alternative, is to add a completely different one.